providers/Flipt/composer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "open-feature/flipt-provider",
+    "description": "The flipt.io provider package for open-feature",
+    "keywords": [
+        "flipt",
+        "open-feature",
+        "feature flags"
+    ],
+    "homepage": "https://github.com/legoheld/flipt-provider",
+    "require": {
+        "php": ">=8.0",
+        "open-feature/sdk": "^2.0",
+        "flipt-io/flipt": "^0.0.1",
```

Comment:
just released `0.2.0` btw https://github.com/flipt-io/flipt-php/releases/tag/v0.2.0

Reply:
@markphelps Updated package to use ResponseConstants from client.

PS: Sorry about the missing conventional commit ;)